lib/gosu.rb

```diff
@@ -1,82 +1,111 @@
 require "ffi"
 require "fiddle"
 
-require_relative "gosu/numeric"
-
-require_relative "gosu/version"
-require_relative "gosu/constants"
-require_relative "gosu/window"
-require_relative "gosu/image"
-require_relative "gosu/font"
-require_relative "gosu/color"
-require_relative "gosu/text_input"
-require_relative "gosu/gl_tex_info"
-require_relative "gosu/channel"
-require_relative "gosu/sample"
-require_relative "gosu/song"
-
-require_relative "gosu/compat"
+if RUBY_PLATFORM =~ /mswin$|mingw32|mingw64|win32\-|\-win32/
+  binary_path = File.expand_path("../../../gosu/lib", __dir__)#File.dirname(__FILE__)
```

Comment:
This hard coded path needs a `../` removed since it's been moved "up" a directory.